client/src/components/squids/SquidForm.jsx

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable no-nested-ternary */
+import React, { useState } from "react";
+
+import { useMutation, queryCache } from "react-query";
+
+import { ApiClient } from "../../backend/ApiClient";
+
+import "../../style/squids/squidForm.pcss";
+
+import { useForm } from "react-hook-form";
+
+import { FormError } from "../common/FormError";
+import { usePostSquidMutation } from "../hooks/usePostSquidMutation";
+
+export const SquidForm = ({ setRefreshMessage, setFormSuccess, formSuccess }) => {
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+    reset,
+  } = useForm({
+    defaultValues: {
+      experiencePoints: 0,
+    },
+  });
+  const postSquidMutation = usePostSquidMutation({ setRefreshMessage, reset });
+
+  const onSubmit = async (data) => {
+    postSquidMutation.mutate(data);
+    setFormSuccess(true);
+  };
+
+  const specialPowerSelectOptions = [
```

Comment:
I'd suggest DRYing this up by `map`ing over an array of values used for both value and label... 

const specialPowerSelectOptions = [
  { value: null, label: "" }, 
  ...["your", "options"].map(...
];